sites/all/modules/custom/language_selector_page/language_selector_page.module

```diff
@@ -121,10 +121,8 @@ function language_selector_page_preprocess_block(&$variables) {
     if (!empty($variables['elements']['other']['#markup'])) {
       foreach ($variables['elements']['other']['#markup'] as $code => $lang) {
         $options = array(
-          'language' => $lang,
```

Comment:
Hello,

It has been removed because this will also change the interface language.

So when the page is: 
*page_en*
and you click in the page level switcher on *dutch* the expected value is:
*page_en?2nd_language=nl*
but instead, if you have this line the result is:
*page_nl?2nd_language=nl*

The page level switcher should leave the interface untouched